packages/forms/src/styled/range/StyledSliderThumb.ts

```diff
@@ -38,16 +42,16 @@ const colorStyles = (props: ThemeProps<DefaultTheme>) => {
     box-shadow: ${boxShadow};
     background-color: ${backgroundColor};
 
-    &[data-garden-focus-visible='true'] {
-      box-shadow: ${focusBoxShadow};
-    }
-
     &:hover,
     &[data-garden-hover='true'] {
       border-color: ${hoverBorderColor};
       background-color: ${hoverBackgroundColor};
     }
 
+    ${focusStyles({
+      theme: props.theme
+    })}
```

Comment:
Destructuring `theme` would condense further.

```suggestion
    ${focusStyles({ theme: props.theme })}
```